build.py

```diff
@@ -51,3 +61,33 @@ def set_properties(project):
         'Programming Language :: Python',
         'Topic :: System :: Networking'
     ])
+
+
+def check_sh(logger):
+    """ Check if 'sh' module is installed'. """
+    if not sh:
+        logger.error("The 'sh' module was not found!")
+        logger.error("Run 'pyb install_dependencies' to enable this task.")
+        raise BuildFailedException('Unable to run task!')
+
+
+def docker_execute(command_list):
+    """ Run and tail a docker command. """
+    running_command = sh.docker(command_list)
+    for line in running_command:
+        print(line.strip())
+
+
+@task
+def docker_build(logger):
+    check_sh(logger)
+    logger.info("Will now attempt to build the docker image.")
+    docker_execute(['build', '-t',
+                    '{0}/{1}:{2}'.format(org_name, name, version), '.'])
```

Comment:
It would be cleaner to accept `project` in the task and then use `project.org_name` and so forth.
Top level locals are automatically bound to the project object and this would allow you to pull out the tasks in a plugin or a "bldsup" module without breaking anything.

Hooray for lose coupling :-P